docs/microbit_micropython_api.rst

```diff
@@ -70,10 +73,35 @@ The LED display is exposed via the `display` object::
     # written messages).
     display.scroll(string, delay=400)
 
+Microphone **V2**
+----------
+
+This Microphone is accessed via the `microphone` object::
+
+    # Value to represent loud sound events, like clapping or shouting.
+    SoundEvent.LOUD = "loud"
+    # Value to represent quiet sound events, like speaking or background music.
+    SoundEvent.QUIET = "quiet"
+    # The name of the last recorded sound event, `loud` or `quiet`.
+    current_sound()
+    # A sound event,  such as `SoundEvent.LOUD` or `SoundEvent.QUIET`. 
+    # Returns`true` if sound was heard at least once since the last
+    # call, otherwise `false`.
+    was_sound(event)
+    # A tuple of the event history. The most recent is listed last.
+    # Also clears the sound event history before returning.
+    get_sounds()
+    # The threshold level in the range 0-255. For example,
+    # `set_threshold(SoundEvent.LOUD, 250)` will only trigger if the
+    # sound is very loud (>= 250).
+    set_threshold()
```

Comment:
this example is empty

Reply:
Fixed in https://github.com/bbcmicrobit/micropython/pull/692/commits/78bd2418a5e2b22c538b233e3313806f2dc032e5